manifests/database.pp

```diff
@@ -28,9 +29,14 @@
     Class['postgresql::server::service'] ~> Service['pulpcore-content.service']
   }
 
+  $migrate_unless = $always_run_migrations ? {
```

Comment:
Perhaps this is a good place to leave a comment that `--check` is broken and why this hack exists

Reply:
Done